### DISCUSSION TITLE
教程：worker store路径配置功能

### DISCUSSION THREAD

**Discussion by Fatman13** [filecoin-project/venus#4675] [Tutorials]
原文在[这里](https://filecoinproject.slack.com/archives/C028PCH8L31/p1641376476290100?thread_ts=1641352580.282200&cid=C028PCH8L31) by @diwufeiwen 

## 场景1: 

`Branch: feat/task_process_opt: 5138bd91e747d6dc2485adf9e9f5ebb262376241`

1. venus-sealer: 192.168.200.8,设置store;
2. venus-worker: 192.168.200.24,不设置store; 
3. 存储机器: 192.168.200.31,挂载到sealer和worker, 挂载路径: /storage-nfs-2/***/

![image](https://user-images.githubusercontent.com/1591330/148314806-3c6be0c8-a341-439d-a205-8af9e7162a98.png)

![image](https://user-images.githubusercontent.com/1591330/148314854-2107435f-1241-4eb9-b149-9d3382884925.png)

venus-sealer config.toml监听地址配置:
```
[API]
  ListenAddress = "/ip4/0.0.0.0/tcp/2345/http"
  RemoteListenAddress = "192.168.200.8:2345"
```
### 测试结果

![image](https://user-images.githubusercontent.com/1591330/148314923-dc5cd4d1-cccf-42a2-bb75-05ebd182b141.png)

从图中可以看出,文件拷贝到永久目录时worker有出口流量,sealer同时存在出入流量.

## 场景2:

`Branch: feat/task_process_opt: 1bed4ec2141de21228d019bd57ca1e813bf56f68`

1. venus-sealer: 192.168.200.8,设置store;
2. venus-worker: 192.168.200.24,设置store,和venus-sealer都指向存储机器目录; 
3. 存储机器: 192.168.200.31,挂载到sealer和worker, 挂载路径: /storage-nfs-2/***/

![image](https://user-images.githubusercontent.com/1591330/148315025-49f8eee4-a704-425a-8ceb-f157104de7cd.png)

![image](https://user-images.githubusercontent.com/1591330/148315043-0f5561f4-ed8c-4cb4-9408-c2af8405aa27.png)

![image](https://user-images.githubusercontent.com/1591330/148315067-4236b89d-afc3-4e3e-8c89-6839ac47977c.png)

从图中可以看出,文件拷贝到永久目录时worker有出口流量,sealer没有流量.此方案需要woker也要设置store路径.

具体代码见Branch中最后一次提交

**Comment by diwufeiwen** [comment]
时机成熟了会整理文档到venus-docs

**Reply by AllynSheep** [reply]
整理了吗？
